Potatso/Core/CloudSetManager.swift

```diff
@@ -30,18 +30,18 @@ class CloudSetManager {
                 uuidsArray.append(tObj as String)
             })
             
-            API.updateRuleSetListDetail(uuidsArray) { (response) in
-                if let sets = response.result.value {
-                    do {
-                        try RuleSet.addRemoteArray(sets)
-                    }catch {
-                        error.log("Unable to save updated rulesets")
-                        return
-                    }
-                }else {
-                    response.result.error?.log("Fail to update ruleset details")
-                }
-            }
+//            API.updateRuleSetListDetail(uuidsArray) { (response) in
```

Comment:
Just note, if you don't need any code, just remove it, no need to comment them out like this.

But we should include code in `CloudSetManager` as for later we will do #62

Reply:
OK. Rarely used git. We use svn in out company. This is my first pull requests in github. it make some mistake.